pkgs/by-name/ya/yandex-music/update.sh

```diff
@@ -19,8 +20,23 @@ date=$(jq -r '.created_at' <<<"$latest_release")
 # truncate time
 date=${date%T*}
 
+importTree="(let tree = import ./.; in if builtins.isFunction tree then tree {} else tree)"
+
+oldVersion=$(nix-instantiate --eval -E "with $importTree; $pname.version" | tr -d '"')
+oldVersionRc="${oldVersion##*rc}"
+if [ "$oldVersionRc" == "$oldVersion" ]; then
+  oldVersionRc="0"
+fi
+oldVersion="${oldVersion%%rc*}"
+
+rc=""
+
+if [ "$version" == "$oldVersion" ]; then
+  rc="rc$((oldVersionRc + 1))";
+fi
```

Comment:
please document what these bash variable transforms are used for. The `##*` and `%%*` are difficult to remember.

Reply:
Done!